translation-web/src/main/resources/translation.properties

```diff
@@ -14,3 +14,11 @@ auth.write.enabled=true
 
 # preffer http over the grpc protocol
 translation.google.usehttpclient=false
+
+translation.eTranslation.baseUrl=https://webgate.ec.europa.eu/etranslation/si/translate
```

Comment:
better move to user.properties it should be updated without creating new release